code/game/objects/structures/window.dm

```diff
@@ -755,8 +755,8 @@
 	rad_insulation = RAD_NO_INSULATION
 	explosion_block = 3
 	glass_type = /obj/item/stack/sheet/plastitaniumglass
-	smoothing_groups = list(SMOOTH_GROUP_SHUTTLE_PARTS, SMOOTH_GROUP_WINDOW_FULLTILE_PLASTITANIUM, SMOOTH_GROUP_PLASTITANIUM_WALLS)
```

Comment:
If you are going to include changes like this, please at least show them instead of lumping them in with another change with generic wording in the PR.

Reply:
While I agree that I should have documented that more clearly (and a lot of other things that the PR changed, as holy crap it was a big one) that does not make it okay to do so now while pointing to my mistake as a reason. The smoothing refactor PR was a massive and complex PR that touched on basically every turf in the game, some things were bound to fall through the cracks.

You are supposedly touching one turf type. If you do want to change other things that is fine, but you really should document said change. Especially as it affects more than just shuttles, it touches on the entire syndicate base and how their windows smooth.

I'm not here to fight with you, I'm just saying please clearly document all the changes you make.

Reply:
I did say I disabled smoothing for shuttle walls on windows in my PR, I meant all shuttle windows, I didn't include a picture because it's the same effect as seen on the main shuttle.

I'll make it a little more now though.